Revert "Find Mode TrustVerify (#3604)"

JonathanLichtnerAMD opened the pull request:
This reverts commit 076e2cb62ecce4005ebd242a70d235cd784f39cc.

Also added a change to re-disable a couple of tests that were readded as part of the "Rework KTN gtest" PR, since these break gfx908 CI.

JonathanLichtnerAMD committed:
Revert "Find Mode TrustVerify (#3604)"

This reverts commit 076e2cb62ecce4005ebd242a70d235cd784f39cc.

JonathanLichtnerAMD committed:
Merge branch 'develop' into jlichtne/revert-pr3604

randyspauldingamd commented:
I'm surprised by this since he had a green CI run, then only tiny changes afterwards. There might be a quick fix instead of reverting immediately. I'll take a look real quick.

JonathanLichtnerAMD commented:
> I'm surprised by this since he had a green CI run, then only tiny changes afterwards. There might be a quick fix instead of reverting immediately. I'll take a look real quick.

Yes, but we don't usually test MI300 in CI, but we might have done that.

Anyway, my revert hit a different CI issue, so I will investigate that while you are looking for a fix.

randyspauldingamd reviewed (approved):
This did not repro, but I can't spend more than the few minutes on it that I have. Revert away!

JonathanLichtnerAMD committed:
Revert "MI300 tuning update (#3875)"

This reverts commit 801eab7f9aeb2659ab2c8de1e51d4bad952c06ae.

randyspauldingamd reviewed (changes requested):
Uh oh--I didn't notice the db files here. We have to be super careful with lfs files. It would be much easier if we can revert the code without modifying the db's. Is that possible?

JonathanLichtnerAMD committed:
Merge branch 'develop' into jlichtne/revert-pr3604

JonathanLichtnerAMD committed:
Revert "Revert "MI300 tuning update (#3875)""

This reverts commit e24f48be55a1259edb542c2143f343837e55ddab.

JonathanLichtnerAMD commented:
I've rebased since I think we have a workaround for the broken CI issue that was blocking us from fixing the other breaking CI issue!

I've also reverted the revert of the "MI300 tuning update" commit since I had reverted that assuming the other CI break was caused by one of the other three changes... but that wasn't the case.

JonathanLichtnerAMD committed:
TMP: workaround jenkins git submodule issue

JonathanLichtnerAMD committed:
TMP: try another workaround...

JonathanLichtnerAMD committed:
Revert "TMP: try another workaround..."

This reverts commit 6b3ff1db1f6392e3344666a133e9a03895bba216.

JonathanLichtnerAMD committed:
Revert "TMP: workaround jenkins git submodule issue"

This reverts commit 67a70f72985406e30edcbd637d51ddc0dc47d7d0.

JonathanLichtnerAMD committed:
TMP: try the latest workaround

JonathanLichtnerAMD committed:
Revert "TMP: try the latest workaround"

This reverts commit 98814ee955b4ef199eb9fa73fefa208f031d300c.

JonathanLichtnerAMD committed:
Merge branch 'develop' into jlichtne/revert-pr3604

JonathanLichtnerAMD commented:
> Uh oh--I didn't notice the db files here. We have to be super careful with lfs files. It would be much easier if we can revert the code without modifying the db's. Is that possible?

Hi @randyspauldingamd, I've reverted the revert of the  MI300 tuning update, since I don't think that's needed anymore.

JonathanLichtnerAMD committed:
Re-disable GPU_KernelTuningNetTestConvAsm1x1U tests

These tests break on gfx908.

randyspauldingamd reviewed (approved):
Note, I haven't checked the revert, just wanted to be sure the db's didn't get out of sync again :)

JonathanLichtnerAMD committed:
Fix a cppcheck error in driver/util_driver.hpp

JonathanLichtnerAMD commented:
I finally got a CI pass, so this can be merged.

It's a long story, but CI appears to be broken on this branch.  However, CI on an indentical branch just passed:
- http://micimaster.amd.com/job/MLLIBS/job/MIOpen/job/jlichtne%252Frevert-pr3604-v2/

These branches are identical:
```
jlichtne@rocm-framework-30:~/MIOpen$ git diff origin/jlichtne/revert-pr3604-v2 origin/jlichtne/revert-pr3604
jlichtne@rocm-framework-30:~/MIOpen$
```

JonathanLichtnerAMD committed:
Merge branch 'develop' into jlichtne/revert-pr3604
